ui/lib/core/addon/components/ttl-picker2.js

```diff
@@ -100,6 +107,7 @@ export default Component.extend({
       seconds,
       timeString: time + unit,
     };
+    console.log({ ttl });
```

Comment:
did you want to keep this console in there?

Reply:
Nope! 😄